libs/common/setup.go

```diff
@@ -16,6 +17,7 @@ import (
 var (
 	Mode                    string // Mode is set in Setup()
 	InsecureFakeTokenEnable = false
+	OrganizationID          *uuid.UUID
```

Comment:
```suggestion
	InstanceOrganizationID          *uuid.UUID
```
or something else, but just `OrganizationID` is a) confusing and b) will cause a great deal of mismatch bugs with `organizationID` variables